fermilibpluginpyscf/_run_pyscf.py

```diff
@@ -0,0 +1,255 @@
+        verbose: Boolean whether to print calculation results to screen.
+
+    Returns:
+        molecule: The updated MolecularData object.
+    """
+    # Prepare pyscf molecule.
+    pyscf_molecule = prepare_pyscf_molecule(molecule)
+    molecule.n_orbitals = int(pyscf_molecule.nbas)
+    molecule.n_qubits = 2 * molecule.n_orbitals
+    molecule.nuclear_repulsion = float(pyscf.gto.energy_nuc(pyscf_molecule))
+
+    # Run SCF.
+    pyscf_scf = compute_scf(pyscf_molecule)
+    pyscf_scf.verbose = 0
+    molecule.hf_energy = float(pyscf_scf.kernel())
+    if verbose:
+        print('Hartree-Fock energy for {} ({} electrons) is {}.'.format(
+            molecule.name, molecule.n_electrons, molecule.hf_energy))
+
+    # Populate fields.
+    molecule.canonical_orbitals = pyscf_scf.mo_coeff.astype(float)
+    molecule.orbital_energies = pyscf_scf.mo_energy.astype(float)
+
+    # Get integrals.
+    one_body_integrals, two_body_integrals = compute_integrals(
+        pyscf_molecule, pyscf_scf)
+    molecule.one_body_integrals = one_body_integrals
+    molecule.two_body_integrals = two_body_integrals
+
+    # Run MP2.
+    if run_mp2:
+        pyscf_mp2 = pyscf.mp.MP2(pyscf_scf)
+        pyscf_mp2.verbose = 0
+        molecule.mp2_energy = molecule.hf_energy + pyscf_mp2.kernel()[0]
+        if verbose:
+            print('MP2 energy for {} ({} electrons) is {}.'.format(
+                molecule.name, molecule.n_electrons, molecule.mp2_energy))
+
+    # Disable non-singlets for now for CI and CC due to not being implemented
+    if (run_cisd or run_ccsd) and molecule.multiplicity != 1:
+        print("WARNING: CISD and CCSD not implemented for "
+              "non-singlet states in PySCF.")
+        run_cisd = run_ccsd = False
+
+    # Run CISD.
+    if run_cisd:
+        pyscf_cisd = pyscf.ci.CISD(pyscf_scf)
+        pyscf_cisd.verbose = 0
+        pyscf_cisd.kernel()
+        molecule.cisd_energy = molecule.hf_energy + pyscf_cisd.e_corr
+        if verbose:
+            print('CISD energy for {} ({} electrons) is {}.'.format(
+                molecule.name, molecule.n_electrons, molecule.cisd_energy))
+
+    # Run CCSD.
+    if run_ccsd:
+        pyscf_ccsd = pyscf.cc.CCSD(pyscf_scf)
+        pyscf_ccsd.verbose = 0
+        pyscf_ccsd.kernel()
+        molecule.ccsd_energy = molecule.hf_energy + pyscf_ccsd.e_corr
+        if verbose:
+            print('CCSD energy for {} ({} electrons) is {}.'.format(
+                molecule.name, molecule.n_electrons, molecule.ccsd_energy))
+
+    # Run FCI.
+    if run_fci:
+        pyscf_fci = pyscf.fci.FCI(pyscf_molecule, pyscf_scf.mo_coeff)
+        pyscf_fci.verbose = 0
+        molecule.fci_energy = pyscf_fci.kernel()[0]
+        if verbose:
+            print('FCI energy for {} ({} electrons) is {}.'.format(
+                molecule.name, molecule.n_electrons, molecule.fci_energy))
+
+    # Return updated molecule instance.
+    molecule.save()
+    return molecule
+
+
+# Test.
```

Comment:
We cannot really test pyscf without setting it up with coveralls and travis. So I just removed this code.